src/AssetManager.Web/Startup.cs

```diff
@@ -55,13 +72,13 @@ public void ConfigureProductionServices(IServiceCollection services)
         }
 
         // This method gets called by the runtime. Use this method to add services to the container.
-        public void ConfigureServices(IServiceCollection services)
-        {
-            services.AddScoped(typeof(IAsyncRepository<>), typeof(EfRepository<>));
+        //public void ConfigureServices(IServiceCollection services)
```

Comment:
Always remember,
> Never comment out any codes
It generate code smells. Everyone eventually forget why the code is commented out after few weeks.